packages/expo-file-system/android/src/main/java/expo/modules/filesystem/next/FileSystemNextDirectory.kt

```diff
@@ -0,0 +1,13 @@
+package expo.modules.filesystem.next
+
+import java.io.File
+
+class FileSystemNextDirectory(file: File) : FileSystemNextPath(file) {
```

Comment:
I would recommend not to use `next` everywhere – I'm pretty sure it won't cause name collisions and you save some later work.

Reply:
Removing this causes some collisions, but I'll keep it only where necessary.